### PR TITLE
Add provisioner for Aurora Postgres, single instance

### PR DESCRIPTION
**Pull request by epavlova**
# Description

## What

We need a provisioner for the new enriched content store. I added specific CloudFormation template for the enriched content store. However I kept the rest of the provisioner generic for any Aurora Postgres in  eu-west-1.

## Why

https://financialtimes.atlassian.net/browse/UPPSF-2357

## Anything, in particular, you'd like to highlight to reviewers

_This provisioner is almost the same as the one for the Ontotext Backup DB. We could move the Ontotext Backup CloudFormation template to this provisioner if the approach is approved. We could keep them separated as well having in mind that we may decommission the Ontotext Backup in the next year._

## Scope and particulars of this PR (Please tick all that apply)

- [ ] Tech hygiene (dependency updating & other tech debt)
- [ ] Bug fix
- [X] Feature
- [ ] Documentation
- [ ] Breaking change
- [ ] Minor change (e.g. fixing a typo, adding config)

___
This Pull Request follows the rules described in our [Pull Requests Guide](https://github.com/Financial-Times/upp-docs/tree/master/guides/pr-guide)

### CODE REVIEW

File: upp-aurora-postgres-provisioner/README.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+export AWS_SECRET_ACCESS_KEY="xxx"
+
+# Fill in the CloudFormation template name that you want to use
+export CF_TEMPLATE="xxx"
+
+# Fill in the database name that you want to use
+export DB_NAME="xxx"
+
+# Specify the DB password
+export MASTER_PASSWORD="xxx" \
```
Comment (karsov) [inline upp-aurora-postgres-provisioner/README.md:27]: There is a typo in the other README too.

```suggestion
export MASTER_PASSWORD="xxx"
```

**Review by karsov — COMMENTED** [review]
The runbook in this PR won't be applied before https://github.com/Financial-Times/upp-provisioners/pull/268 is merged.

**Comment by karsov** [comment]
Maybe it would be better if we move the other provisioner here.

**Comment by epavlova** [comment]
> Maybe it would be better if we move the other provisioner here.

Okay, I will do that as part of a separate PR. I don't want to potentially break two provisioners/DB with one shot :). 
Btw - it is tested that one is able to connect to the dev DB via the restricted VPN and also it is tested that a service running on our Dev clusters can access the DB.

**Review by bboykov — APPROVED** [review]

**Review by dbelev — APPROVED** [review]

**Review by karsov — APPROVED** [review]

**Pull request merged**